nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-api/src/main/java/org/nuxeo/ecm/directory/BaseDirectoryDescriptor.java

```diff
@@ -220,6 +252,29 @@ public String getCreateTablePolicy() {
         return ctp;
     }
 
+    /**
+     * @since 11.1
+     */
+    @XNode("dataLoadingPolicy")
+    public String dataLoadingPolicy;
+
+    /**
+     * Returns the dataLoadingPolicy; default is {@link #NEVER_LOAD}.
+     *
+     * @since 11.1
+     */
+    public String getDataLoadingPolicy() {
+        if (StringUtils.isBlank(dataLoadingPolicy)) {
+            return NEVER_LOAD;
+        }
+        String dlp = dataLoadingPolicy.toLowerCase();
```

Comment:
`toLowerCase(Locale.ENGLISH)`.

Because in the Turkish locale, the lowercase version of `I` isn't `i` but `ı` (dotless i) — and there is an uppercase i with dot: `İ`

Reply:
So Should I also cleanUP the old method   `getCreateTablePolicy()` :  which  code : `String ctp = createTablePolicy.toLowerCase();` ?

Reply:
Yes please.